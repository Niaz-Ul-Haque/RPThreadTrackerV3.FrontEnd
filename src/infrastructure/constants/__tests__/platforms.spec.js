import platforms from '../platforms';

describe('object structure', () => {
	it('should have expected values', () => {
		expect(platforms).toMatchSnapshot();
	});
});
