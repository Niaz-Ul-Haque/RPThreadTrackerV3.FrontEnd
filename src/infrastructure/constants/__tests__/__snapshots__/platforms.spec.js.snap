// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`object structure should have expected values 1`] = `
Object {
  "1": "Tumblr",
}
`;
